app/models/order.rb

```diff
@@ -7,4 +7,8 @@ class Order <ApplicationRecord
   def grandtotal
     item_orders.sum('price * quantity')
   end
+
+  def total_items
+    item_orders.sum('quantity')
```

Comment:
This is an excellent example of letting AR and the database do the calculations for you! It's very MVC 👍🏻.